core/src/main/scala/kafka/server/KafkaBroker.scala

```diff
@@ -178,6 +179,8 @@ abstract class KafkaBroker(val config: KafkaConfig,
 
   var kafkaController: KafkaController = null
 
+  var metadataEventManager: BrokerMetadataEventManager = null
```

Comment:
like we were talking about outside of JIRA, how about something like `BrokerMetadataListener` for the name?  Might be a bit shorter if we can keep "Event" out of there.